clang-format.sh

```diff
@@ -14,8 +14,6 @@
 
 # Runs clang-format and show diff.
 
-alias clang-format=clang-format-9
```

Comment:
Why remove this?

Reply:
we've upgraded the clang version and the docker image only contains 1 clang-format version anyways. hence "clang-format" is sufficient.